### ISSUE TITLE
For whatever reason I can't have B2SBackglassServerRegisterApp.exe (either the latest update or any prior) open to the dialogue windows. 

### ISSUE THREAD

**Issue by tabqwerty** [vpinball/b2s-backglass#126]
I just did a system restore on Windows 11 Pro and am now reinstalling this update. I spent the whole day trying to figure out why my backglasses aren't showing, not even in another window with alt+tab. All of the .directb2s files I put into the tables folder didn't associate with the B2S Server EXE until I went into properties and also unblocked them (I don't know why they were all blocked, either). Yes, they all have identical names to the .vpx table files. No, I don't have "disable .b2s" ticked off in VPX settings. 

I'd attach an image of what I see, but it's only the permissions asking if I want to run the file and nothing else after. 

I know this is probably extremely uncommon. Like I said, I have no idea why this is happening to me.

**Comment by JockeJarre** [comment]
When creating an issue, there is a list of things which has to be filled out before pressing "Submit new issue":

> **Describe the bug**
> A clear and concise description of what the bug is.
> 
> *Note:* Are you sure there is a bug in the B2S Server and not anything related to configuration or other dependencies? If you are unsure please create a post on the [Virtual Pinball Chat Discord Server](https://discord.com/invite/YHcBrtT), channel `#db2s` or on [VPUniverse](https://vpuniverse.com/files/file/12024-b2s-backglass-server/) first.

"We" as in "me, myself and I", is not a commercial offering to find **your problems**. Please discuss with others (including me) first as described above.
